### ISSUE TITLE
[Feature Request] Is the "Verified Game Server" widget for store pages planned to be re-implemented?

### ISSUE THREAD

**Issue by Jump-Suit** [discord/gamesdk-and-dispatch#44]
I would like to reintroduce the store page links we game devs are capable of sharing via dev portal, if players view them externally, see that widget upsell may say for them to go there after reviewing the store page information from that external link for the store page and visit the Game Server.

![image](https://user-images.githubusercontent.com/26616362/78690929-ac1daf00-78bd-11ea-98b7-7c34fc90cba1.png)

Found a older version of the widget I speak of.

Suggested by me (JumpSuit) - From Discord GameSDK Help Server.

**Comment by TheGoldenCactus11** [comment]
+1 Yes please

**Comment by Variiuz** [comment]
+1

**Comment by TheGoldenCactus11** [comment]
The code is already on Discord, and shouldn't be hard to add back to all stores

**Comment by Jump-Suit** [comment]
Agreed, it still exists today and should be available to any Store Page if possible, or Public Servers!
